portal/src/main/webapp/js/src/oncoprint/setup.js

```diff
@@ -871,6 +868,14 @@ window.CreateCBioPortalOncoprintWithToolbar = function (ctr_selector, toolbar_se
 	    oncoprint.setSortConfig({'type':'order', 'order':order});
 	};
 	
+	var updateViewMenuLabels = function(state) {
```

Comment:
this doesn't bother me because it's still clear which function is being called. all that may be unclear to the reader who doesn't know about hoisting is how it's possible that its being called before its declared. but I think anyone reading it would just Control+F and find `updateViewMenuLabels`